pages/company/[filename].tsx

```diff
@@ -42,7 +40,7 @@ export default function CompanyPage(
     >
       <div>
         <SEO seo={props.seo} />
-        <Layout menu={data.megamenu}>
+        <Layout menu={data.megamenu} showAzureBanner>
```

Comment:
this will probably have to be undone during the app router migration (i.e. we can't add stuff to the layout in the page renderer)

Reply:
done